release/tag_repos.sh

```diff
@@ -177,6 +182,11 @@ case "${subcmd}" in
 status)
 	check_versions
 	;;
+pre-release)
+	unset repos[2]
```

Comment:
Instead of do this could you try.
```
check_versions $(cat "${GOPATH}/src/${runtime_repo}/VERSION")
...
#$1 Version to check to branches merged
check_versions() {
version_to_check=${1:-}
if version_to_check == "";then
info query the version from latest runtime in branch ${branch}
else
    kata_version=${version_to_check}
fi
...
for repo in "${repos[@]}"; do
  if repo == kata-runtime; then
      not checking runtime because we want the rest of repos are in $version_to_check
      continue
  fi
}
```

Reply:
yep - let's do a specific check or some smart way to 'build up' the list of repos to check, and not add the runtime to the list in the first place if we are doing the pre-check.